_posts/2021-09-29-solidity-0.8.9-release-announcement.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.8.9 Release Announcement'
+date: '2021-09-29'
+author: Solidity Team
+category: Releases
+---
+
+[Solidity v0.8.9](https://github.com/ethereum/solidity/releases/tag/v0.8.9) fixes two bugs:
+
+ - one related to [immutables of signed integer types shorter than 256 bits](TODO LINK)
+ - one introduced with [user defined value types](https://blog.soliditylang.org/2021/09/27/user-defined-value-types/) in Solidity v0.8.8 (released yesterday).
+
+The bug has no influence on the correctness of Solidity contracts, but contracts compiled with
+Solidity 0.8.8 that use the new feature are unnecessarily wasteful and might have problems
+with tooling or contract upgrades.
+
+### Storage Layout of User Defined Value Types
+
+The compiler did not correctly compute the storage layout of user defined value types
+based on types that are shorter than 32 bytes. It would always use a full storage slot
+for these types, even if the underlying type is shorter.
+
+Types shorter than 32 bytes can share a storage slot with other values if there is
+enough space.
+
```

Comment:
I feel this section (`Storage Layout of ...`) may be replaced by a sentence such as "For further details about the individual bugs, please read the linked blog posts." Another reason this section may be removed is because we don't have a separate section for the immutable bug in the release blog post.

Reply:
Yes, we will remove it as soon as we have the blog post about the user defined value types bug.